tests/MongoDB.Driver.TestHelpers/DriverTestConfiguration.cs

```diff
@@ -54,6 +56,14 @@ public static MongoClient Client
             get { return __client.Value; }
         }
 
+        /// <summary>
+        /// Gets the test client with multiple shard routers.
+        /// </summary>
+        public static MongoClient ClientWithMultipleShardRouters
+        {
+            get { return __clientWithMultipleShardRouters.Value; }
```

Comment:
I wanted to merge this property and `Client` itself and always use `Client` that could have `MultipleShardRouters` connection string for sharded server, but it led to errors in other tests so I gave up

Reply:
Right I think there are a lot of tests that assume we are connected to a single `mongos` (if the cluster is sharded).